taxi/tests/test_views.py

```diff
@@ -0,0 +1,63 @@
+from django.test import Client
+from django.urls import reverse
+import pytest
+
+
+from taxi.models import Manufacturer, Car, Driver
```

Comment:
If importing more than two items, it is better to write them on separate lines. This improves readability and maintenance of the code. 

Example:

From taxi.models import ( 
	Driver,
	Manufacturer,
	Car
)
Read more about topic [here](https://peps.python.org/pep-0008/#imports)